indigo_app/templates/indigo_api/_provisions_tree_commenced.html

```diff
@@ -1,20 +1,23 @@
 {% for prov in provisions %}
   {% if prov.commenced or prov.children and not prov.all_descendants_same %}
     <li>
-      <span class="{% if prov.commenced %}text-body{% else %}text-black-50{% endif %}">
-        {% if prov.commenced %}
-          <input type="checkbox" checked disabled/>
-        {% endif %}
-        {{ prov.title|default:prov.id }}
-      </span>
-
-      {% if prov.commenced and prov.all_descendants_same %}
-        (Commenced in full)
-      {% elif prov.children %}
-        <ul class="provisions-list">
-          {% include "indigo_api/_provisions_tree_commenced.html" with provisions=prov.children %}
-        </ul>
+      {% if prov.commenced %}
+        <input type="checkbox" disabled checked/>
```

Comment:
Self-closing tags aren't supported in HTML5.
```suggestion
        <input type="checkbox" disabled checked>
```